sdk/python/Makefile

```diff
@@ -18,23 +18,24 @@ TEST_ALL_DEPS = build
 
 ensure::
 	$(PYTHON) -m venv venv
-	. venv/*/activate && python -m pip install -r requirements.txt
+	. venv/*/activate && pip install wheel
```

Comment:
The change from `python -m pip` to `pip` breaks my environment because it picks up the global pip instead of the venv pip. D'you need the change or it was done for aesthetics?

```
[nix-shell:~/pulumi/sdk/python]$ make ensure
make ensure
ENSURE:
python3 -m venv venv
. venv/*/activate && pip install wheel
Collecting wheel
  Using cached wheel-0.37.1-py2.py3-none-any.whl (35 kB)
Installing collected packages: wheel
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/nix/store/w8lqcsrwlwrvmchj46ph9880w9h4m50a-python3-3.9.6/lib/python3.9/site-packages/wheel'
Consider using the `--user` option or check the permissions.

WARNING: You are using pip version 21.1.3; however, version 21.3.1 is available.
You should consider upgrading via the '/nix/store/w8lqcsrwlwrvmchj46ph9880w9h4m50a-python3-3.9.6/bin/python3.9 -m pip install --upgrade pip' command.
make: *** [Makefile:21: ensure] Error 1

[nix-shell:~/pulumi/sdk/python]$ (. venv/*/activate && which pip)
(. venv/*/activate && which pip)
/nix/store/h5j6wymds31fgj0pdhfrdh27qhlw6d1v-python3.9-pip-21.1.3/bin/pip

```

Reply:
Interesting - how is it picking up your pip over the just-sourced venv? Is that a nix issue perhaps, something in your nix env modifying the path order and taking precedent over the venv?